WcaOnRails/spec/fixtures/files/results.json

```diff
@@ -0,0 +1,3 @@
+{
+  "results": "good"
+}
```

Comment:
Why did this file end up here? Shouldn't it be removed in the `after` spec hooks?

Reply:
No, it's not created every time.
I'm using it in the mailer tests, which only make sure the file is read correctly. So I chose to create it only once and use the same file every time.
But I can change that.